babel/support.py

```diff
@@ -249,6 +255,8 @@ def __delattr__(self, name):
         delattr(self.value, name)
 
     def __getattr__(self, name):
+        if self._attribute_error is not None:
+            raise AttributeError(self._attribute_error)
```

Comment:
Shouldn't this just `raise self._attribute_error`, since you're already stashing the exception itself?

Reply:
Done in last commit. Thanks!